src/util/graphQLFetcher.js

```diff
@@ -34,6 +34,11 @@ export default function graphQLFetcher(lgJWT, baseURL, origin = ORIGIN_URL) {
         }
         return resp.json()
       })
-      .then(graphQLResponse => graphQLResponse.data)
+      .then(({errors, data}) => {
```

Comment:
yeah, it's needed. The API returns a successful error code (200) when we throw a GraphQLError on the server. I figured that was just how graphql worked, since it's not REST and doesn't use http protocol things.